### DISCUSSION TITLE
mybinder pasting .jl into new notebook give errors

### DISCUSSION THREAD

**Discussion by paradocs** [fonsp/Pluto.jl#2334] [Pluto development] [answered]
Start Pluto in mybinder, then "Create a new notebook"
We paste in a previously saved notebook and run it.
1) We get an error in cell 1 because 
	using Markdown
	using InteractiveUtils
do not have       begin           end
Solution: Add begin end  for default using statements.
2) Again with pasting a previously saved notebook:
we must delete the last two cells named
PLUTO_PROJECT_TOML_CONTENTS
PLUTO_MANIFEST_TOML_CONTENTS
I believe that these entries are recreated.
We can have our students delete these cells before
saving but it would be nice if this was automated.

Thank you for making Pluto notebooks a great
teaching tool for easy use at many levels.

**Comment by fonsp** [comment, marked as answer]
Accidental duplicate of https://github.com/JuliaPluto/PlutoTest.jl/discussions/30#discussioncomment-3933431
